test/parallel/test-process-startTime.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const now = Date.now();
+const diffFromNow = now - process.startTime;
+
+// Make it a bit flexible
+assert.ok(diffFromNow > 0);
+assert.ok(diffFromNow < common.platformTimeout(100));
```

Comment:
Ah, gotcha. I can remove it if others agree it'd be too flaky.

And if it's not closed, of course 😄